introspective_ORB_SLAM/include/dataset_creator.h

```diff
@@ -19,50 +19,37 @@
 #ifndef iSLAM_DATASET_CREATOR
 #define iSLAM_DATASET_CREATOR
 
-#include <vector>
-#include <algorithm>
-#include <glog/logging.h>
+#include <opencv2/opencv.hpp>
+#include <jsoncpp/json/json.h>
 #include <Eigen/Core>
-#include <Eigen/Geometry>
-#include <glog/logging.h>
 
-#include <iostream>
-#include <fstream>
+#include <vector>
 #include <string>
-#include <sstream>
-#include <sys/stat.h>
-#include <dirent.h>
-#include <jsoncpp/json/json.h>
-
-#include "opencv2/video/tracking.hpp"
-#include "opencv2/imgproc/imgproc.hpp"
-#include "opencv2/highgui/highgui.hpp"
-#include "opencv2/features2d/features2d.hpp"
-#include "opencv2/calib3d/calib3d.hpp"
-#include "opencv2/features2d.hpp"
-// #include "opencv2/xfeatures2d.hpp"
-
-#include "io_access.h"
-
```

Comment:
Was there a reason all these headers were included? Am I missing something major and making a mistake by removing them?

Reply:
Nope, just dead code from removed features.